### PR TITLE
Stop using old s3 filename convention

### PR DESCRIPTION
**Pull request by lowellnava**
Connects: #583. Know all files with old filenaming convention have expired, will monitor when pushed live.

### CODE REVIEW

File: spec/models/document_spec.rb [hunk 1]
```diff
@@ -61,22 +61,12 @@
     subject { document.s3_filename }
 
     let(:document) do
-      Document.new(id: 5, document_id: "{TEST}", mime_type: "application/pdf", download_id: 45)
```
Comment (mdbenjam) [inline spec/models/document_spec.rb]: Why was this removed?

**Comment by mdbenjam** [comment]
I just checked in our S3 bucket and it looks like everything is named with the new convention.

File: spec/models/document_spec.rb (hunk 1 above)
Comment (lowellnava) [reply spec/models/document_spec.rb]: I removed it because I didn't think it would be used anywhere after old_s3_filename is removed. `grep -r download_id caseflow-efolder | grep -v test.log` looks like it may still exist in db/schema.rb though so maybe I was a little hasty to remove this attribute. (

Comment (lowellnava) [reply spec/models/document_spec.rb]: Just chatted with @mdbenjam and this is some rails magic that I didn't understand. Adding it back in.
